docs/overview/script-storage.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Office Scripts file storage and ownership'
+description: 'Information about how Office Scripts are stored in OneDrive and transferred between owners.'
+ms.date: 11/13/2020
+localization_priority: Normal
+---
+
+# Office Scripts file storage and ownership
+
+Office Scripts are stored as **.osts** files in your OneDrive. This allows your scripts to exist outside any particular workbook. Your OneDrive settings control the shared access and permissions for these script **.osts** files; independent of any Excel settings.
+
+## File storage
+
+You Office Scripts are stored in your OneDrive. The **.osts** files are found in the **/Documents/Office Scripts/** folder. Any edits made these **.osts** files, such as renaming or deleting files, will be reflected in the Code Editor and Script Gallery.
+
+Scripts that are shared with one of your workbooks remain in the cloud and are not copied to any of your local or OneDrive folders. The **Make a Copy** button of the Code Editor saves a separate copy of the script in your OneDrive. Changes to the copy don't affect the original script.
```

Comment:
It's a little ambiguous to me what the difference is between a script "in the cloud" and "in your OneDrive". Is there another cloud file system option other than OneDrive?

Reply:
No, it's all OneDrive, but it centers around which OneDrive is being used. How about:
"Scripts that are shared with one of your workbooks remain in the script creator's OneDrive. They are not copied to any of your local or OneDrive folders when you run the shared script in Excel."